### ISSUE TITLE
Android performance - is it terrible? 

### ISSUE THREAD

**Issue by floatplane** [NWACus/avy#522]
It seems terrible on my test phone, not sure how underpowered it is

**Comment by floatplane** [comment]
I am experiencing huge lag at startup - the app renders, but is completely unresponsive for about 15 seconds. Android perf monitor on the phone shows us rendering at a steady 60fps so I have no idea what the app is doing during this period!

**Comment by floatplane** [comment]
seems generally better after #537 and #539 and #540. I still occasionally see weird input lag at startup on my Android device but I'm not sure if that's widespread, and if so it maybe should be a separate issue
